tests/helper/mongoDB.js

```diff
@@ -0,0 +1,136 @@
+const { MongoClient } = require('mongodb');
+
+class MongoDB extends Helper {
```

Comment:
By the name, it's a connection to the daemon, which it is not.
If it interfaces with the shell, should we give it a more explicit name, i.e. `class MongoDBShell`?
Up to you anyway )

Reply:
Thanks I've changed it to `MongoDBHelper`